scripts/apidocgen/markdown-template/README.md

```diff
@@ -5,60 +5,60 @@ Swagger / OpenAPI 2.0 definitions are converted automatically.
 
 ### Code templates
 
-* `method` - the HTTP method of the operation (in lower-case)
-* `methodUpper` - the HTTP method of the operation (in upper-case)
-* `url` - the full URL of the operation (including protocol and host)
-* `consumes[]` - an array of MIME-types the operation consumes
-* `produces[]` - an array of MIME-types the operation produces
-* `operation` - the current operation object
-* `operationId` - the current operation id
-* `opName` - the operationId if set, otherwise the method + path
-* `tags[]` - the full list of tags applying to the operation
-* `security` - the security definitions applying to the operation
-* `resource` - the current tag/path object
-* `parameters[]` - an array of parameters for the operation (see below)
-* `queryString` - an example queryString, urlEncoded
-* `requiredQueryString` - an example queryString for `required:true` parameters
-* `queryParameters[]` - a subset of `parameters` that are `in:query`
-* `requiredParameters[]` - a subset of `queryParameters` that are `required:true`
-* `headerParameters[]` - a subset of `parameters` that are `in:header`
-* `allHeaders[]` - a concatenation of `headerParameters` and pseudo-parameters `Accept` and `Content-Type`, and optionally `Authorization` (the latter has an `isAuth` boolean property set true so it can be omitted in templates if desired
+- `method` - the HTTP method of the operation (in lower-case)
```

Comment:
To be honest, you can delete this file. If I need to take a look at available properties, I'm using `data.utils.inspect(data)` instead.

Reply:
Alright, fixed! 👍🏻